fa25-team-a/analysis/resale_price_original_workflow/ParsePrices_OriginalWorkFlow.ipynb

```
@@ -0,0 +1,192 @@
+        "            if re.search(pattern, text, flags=re.IGNORECASE):\n",
+        "                return town\n",
+        "        return ''\n",
+        "    df['Town'] = df['Clean_Property'].apply(find_town)\n",
+        "    return df\n",
+        "\n",
+        "\n",
+        "# --- Function 6: Clean address field ---\n",
+        "def clean_address_field(df):\n",
+        "    \"\"\"Remove town names and unit references from the property string.\"\"\"\n",
+        "    def clean_address(text, town, unit):\n",
+        "        if town:\n",
```

Ruff (0.14.8)

Undefined name `display`

(F821)